governance-exercises/Le Bateleur - Poll Exercise 01/Forum Response.md

```diff
@@ -0,0 +1,45 @@
+# Poll Assignment 01
+
+### My Forum Responses
+
+---
+
+Response #1
+
+```
+Author:  Le Bateleur
+Date: Wednesday 19th July 2023
+```
+
+Hi @Alice, @Faraz & @Bob
+
+I'd like to ensure that Maker governance has access to sufficient information to make an informed vote, and that there is optimal clarity and transparency around the polling process generally. 
+
+First: @Alice, I want to confirm that you are aware of, and in agreement with, Faraz/advisors' proposal to adjust the Spork D3M Debt Increase Cooldown (`ttl`) parameter to 86,400 seconds (24 hours).
```

Comment:
It's worth also explaining why you're asking for this. You explained it in the course of action, I think, but worth communicating it to Alice as well. Something like: 'Although you indicated general agreement, I want to clarify you're fine with this specific value' or something.